Core/src/IfcGeometryConverter/GeometrySettings.h

```diff
@@ -175,25 +175,62 @@ namespace OpenInfraPlatform
 				{
 					return abs(first - second) < precision;
 				}
-				/*!Compare two 2D vectors for equality to the specified precision
+
+				/*!Compare two doubles for equality to the default precision
+				*\param[in]    first      The first value
+				*\param[in]    second     The second value
+
+				*\return true, if the absolute difference is smaller than the precision. False otherwise
+				*/
+				bool areEqual(const double first, const double second) const
+				{
+					return abs(first - second) < getPrecision();
+				}
+
+				/*!Compare two 2D vectors for equality to the default precision
 				*\param[in]    first      The first 2D-point of vector
 				*\param[in]    second     The second 2D-point of vector
 
 				*\return true, if the absolute difference is smaller than the precision. False otherwise
 				*/
 				bool areEqual(const carve::geom::vector<2>& first, const  carve::geom::vector<2>& second) const
 				{
-					return areEqual(first.x, second.x, getPrecision()) && areEqual(first.y, second.y, getPrecision());
+					return areEqual(first.x, second.x) && areEqual(first.y, second.y);
 				}
-				/*!Compare two 3D vectors for equality to the specified precision
+
+				/*!Compare two 3D vectors for equality to the default precision
 				*\param[in]    first      The first 3D-point of vector
 				*\param[in]    second     The second 3D-point of vector
 
 				*\return true, if the absolute difference is smaller than the precision. False otherwise
 				*/
 				bool areEqual(const  carve::geom::vector<3>& first, const  carve::geom::vector<3>& second) const
 				{
-					return areEqual(first.x, second.x, getPrecision()) && areEqual(first.y, second.y, getPrecision()) && areEqual(first.y, second.z, getPrecision());
+					return areEqual(first.x, second.x) && areEqual(first.y, second.y) && areEqual(first.y, second.z);

```

Comment:
Here too, call the other overload taking two vectors, with the additional parameter being `getPrecision()`.

Reply:
Resolved: 4a44505